tests/testthat.R

```diff
@@ -1,8 +1,9 @@
 library(testthat)
 library(baselinenowcast)
+library(tidyr)
```

Comment:
Rather than introducing this as a test dep just manually have the long format data in the test. This makes it easier to read/check it is okay and more stable as it doesn't introduce a hidden dep on tidyr

Reply:
removed tidyr dependency and replaced with the result itself in the test